packages/react/src/ImageSlider/ImageSlider.test.tsx

```diff
@@ -22,35 +20,52 @@ window.IntersectionObserver = jest.fn(() => ({
 }))
 
 describe('Image slider', () => {
+  const slides: SlideProps[] = [
```

Comment:
These are images. Let’s create a good distinction between Slides and Images – so rename ‘slide’ to ‘image’ where possible.

Reply:
A `ImageSliderItem` is a wrapper for a image, and i would like to prevent naming it "ImageSliderImageProps", the parent is called a "Slider" and a child would be a "Slide"? Which is also weird, so I settled on "Item".

Looking at the naming and hierarchy I would suggest using:

- ImageSlider
  - ImageSliderScroller
    - ImageSliderItem
        - Image (import) 
  - ImageSliderThumbnails
      - ImageSliderThumbnail 

I tried to be consistent in naming, using the component name a prefix for its children (excluding imported children). I'm not a fan of diving in and renaming one thing because of semantics.

Or rename the whole thing to what it is, a Carousel. [Here is an article](https://www.smashingmagazine.com/2023/02/guide-building-accessible-carousels/) on a carousel but they name the children "Slide".

Reply:
It’s not a Carousel – it has no text content and doesn’t automatically slide.

`ImageSliderItem` is fine.